Update 3d-models folder and FAQ (#137)
vircadia/vircadia-docs-sphinx: docs/source/faq.rst

| The Vircadia folder name might be different on development or testing builds.                       |
| E.g. ``Vircadia - dev``.                                                                            |
+-----------------------------------------------------------------------------------------------------+

+-----------------------------------------------------------------------------------------------------+
| .. _file-manager-help:                                                                              |
|                                                                                                     |
| How do I access the hidden folders that most Vircadia files are stored in?                          |
+=====================================================================================================+
| On all currently supported operating systems the folders that Vircadia stores most of its data in   |
| are hidden.                                                                                         |
|                                                                                                     |

> | The easiest was to access them is to just copy the path and paste into your systems file manager.   |

Reviewer: ```suggestion
| The easiest was to access them is to just copy the path and paste it into your systems file manager.|
```

Author: ```suggestion
| The easiest way to access them is to just copy the path and paste it into your systems file manager.|
```

| Most (if not all) of them have a bar at the top.                                                    |
|                                                                                                     |
| .. image:: _images/file-manager-path.png                                                            |
|                                                                                                     |
| Even if it doesn't look like you can paste anything into the bar, still hover over it and try       |

> | pasting with a shortcut like Ctrl + V                                                               |

Reviewer: ```suggestion
| pasting with a shortcut like Ctrl + V.                                                              |
```
